### ISSUE TITLE
[Bug] perk Doppelganger

### ISSUE THREAD

**Issue by Thirteen0130** [PavelZinchenko/event-horizon-frontier-builds#188]
После смерти второго корабля с перком Doppelganger все дроны и клоны от него сходят с ума и нападают на своих. Сделайте, чтобы дроны и клоны были привязаны к кораблю (то есть на стороне игрока) до 3 смертей второго корабля, как это обычно и происходило

**Comment by Zen9890** [comment]
Нет, такое делать нельзя, можно только как альтернативу сделать чтобы они испарались полностью, иначе перком на респавн можно будет спавнить неограниченное количество клонов из корабля просто делая из него картонку

**Comment by Thirteen0130** [comment]
> Нет, такое делать нельзя, можно только как альтернативу сделать чтобы они испарались полностью, иначе перком на респавн можно будет спавнить неограниченное количество клонов из корабля просто делая из него картонку

Хмм, дааа... Согласен. Перк на респавн здесь всё поломает. Вариант с испарением лучше
